### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by tomwizard-eyes**

### CODE REVIEW

File: docs/resources/agent_to_agent.md [hunk 1]
```diff
@@ -8,57 +8,57 @@ description: |-
...
-
-
+## Example Usage
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `agents` (Block Set, Min: 1) Agents to use (see [below for nested schema](#nestedblock--agents))
```
Comment (raul-te) [inline docs/resources/agent_to_agent.md:20]: Everything under ` <!-- schema generated by tfplugindocs -->` will be overwritten by `go generate`. These descriptions will need to be changed in [schemas.go](https://github.com/thousandeyes/terraform-provider-thousandeyes/blob/496382e168f660e21f5a640f2632658af043506e/thousandeyes/schemas.go).

File: docs/resources/agent_to_server.md [hunk 2]
```diff
@@ -8,19 +8,19 @@ description: |-
...
-
-
+## Example Usage
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `agents` (Block Set, Min: 1) Agents to use (see [below for nested schema](#nestedblock--agents))
```
Comment (raul-te) [inline docs/resources/agent_to_server.md:20]: Everything under ` <!-- schema generated by tfplugindocs -->` will be overwritten by `go generate`. These descriptions will need to be changed in [schemas.go](https://github.com/thousandeyes/terraform-provider-thousandeyes/blob/496382e168f660e21f5a640f2632658af043506e/thousandeyes/schemas.go).

**Review by raul-te — CHANGES_REQUESTED** [review]
A few changes are required for this to work. You need to update the fields description in `schemas.go`, and then run `go generate`. The CI job configured in this repository is failing because of this.

File: thousandeyes/schemas.go [hunk 3]
```diff
@@ -185,232 +213,246 @@ var schemas = map[string]*schema.Schema{
...
-					Description: "Rule ID",
+					Description: "The unique ID of the alert rule.",
 					Optional:    true,
 				},
 			},
 		},
 	},
 	"alert_type": {
-		Description:  "Acceptable test types, verbose names",
+		Description:  "[Page Load, HTTP Server, End-to-End (Server), End-to-End (Agent), DNS+ Domain, DNS+ Server, DNS Server, DNS Trace, DNSSEC, Transactions, Web Transactions, BGP, Path Trace, FTP, SIP Server] The type of alert rule. Acceptable values include the verbose names of supported tests.",
```
[the remark this answers is not in the document]
Comment (tomwizard-eyes) [reply thousandeyes/schemas.go:297]:
```suggestion
		Description:  "[Page Load, HTTP Server, End-to-End (Server), End-to-End (Agent), DNS Server, DNS Trace, DNSSEC, Transactions, Web Transactions, BGP, Path Trace, FTP, SIP Server] The type of alert rule. Acceptable values include the verbose names of supported tests.",
```

**Review by raul-te — DISMISSED** [review]

**Review by raul-te — APPROVED** [review]

**Pull request merged**